contracts/mock/providers/chainlink/InverseChainlinkPairAggregatorMock.sol

```diff
@@ -28,7 +28,7 @@ contract InverseChainlinkPairAggregatorMock is InverseChainlinkPairAggregator {
 
     /** Internal Functions */
     
-    function _normalizeDecimals(int256 value) internal view returns (int256) {
+    function _normalizeDecimals(int256 value) internal pure returns (int256) {
```

Comment:
This function can be removed completely - it's never used anywhere anymore

Reply:
You're right. I removed this contract. Now we are using the real contract in the test.